packages/pwa-buildpack/lib/Utilities/addImgOptMiddleware.js

```diff
@@ -91,35 +85,28 @@ https://github.com/nodejs/node-gyp#installation`
             // height and width.
             //   https://github.com/magento-research/express-sharp/blob/develop/lib/transform.js#L23
             //   https://sharp.pixelplumbing.com/en/stable/api-resize/
-            if (height) {
-                rewrittenUrl += `/${height}`;
-                params.set('crop', true);
+            if (query.width && query.height) {
+                query.crop = true;
             }
 
-            const rewritten = new URL(rewrittenUrl);
-            rewritten.search = params.toString();
-
-            debug({ rewritten });
-
-            // make relative url
-            const url = rewritten.href.slice(rewritten.origin.length);
-
-            // make new query object for express-sharp to use
-            const query = {};
-            for (const [key, value] of params) {
-                query[key] = value;
-            }
-            debug({ url, query });
-            return { url, query };
+            debug('rewrote query to %o', query);
```

Comment:
I noticed we are now leaving `auto: 'webp'` on the `query` object. Is that intended
```
2019-10-07T20:37:31.473Z pwa-buildpack:Utilities:addImgOptMiddleware.js rewrote query to { auto: 'webp', format: 'jpeg', width: '640', height: '800', progressive: true, crop: true }
```